src/Tasks/Common/Resources/Strings.resx

```diff
@@ -401,4 +401,12 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1070: The application configuration file must have root configuration element.</value>
     <comment>{StrBegin="NETSDK1070: "}</comment>
   </data>
+  <data name="AppHostNotWindows" xml:space="preserve">
+    <value>NETSDK1071: Unable to use '{0}' as application host executable because it's not a Windows PE file.</value>
+    <comment>{StrBegin="NETSDK1071: "}</comment>
+  </data>
+  <data name="AppHostNotWindowsCLI" xml:space="preserve">
+    <value>NETSDK1072: Unable to use '{0}' as application host executable because it's not a Windows executable for teh CUI subsystem.</value>
```

Comment:
There is a typo here. teh instead of the. Also, CUI system? Should it be GUI?

Reply:
Thanks, fixed. CUI is the Windows term for console app subsystem. GUI - Graphical subsystem, CUI - Console subsystem. The apphost.exe should be prebuilt as a CUI, if not this check will fail (also there to make sure we didn't compute wrong index).
I added the (Console) into the error message to make this clearer.